napari/_qt/qt_main_window.py

```diff
@@ -459,6 +460,24 @@ def _save_current_window_settings(self):
         if settings.application.save_window_state:
             settings.application.window_state = window_state
 
+    def _warn_on_shimmed_plugins(self) -> None:
+        """Warn about shimmed plugins if needed."""
```

Comment:
```suggestion
        """Warn when shimmed plugins are newly installed, if needed.
        
            In 0.6.0, a plugin using the deprecated plugin engine will have its
            name shimmed so it can be used with npe2. A dialog is displayed
            if there are new shimmed plugins and the warning setting is set to display
            new shimmed plugins.
        """
```